test/functional/test_framework/mininode.py

```diff
@@ -722,6 +722,8 @@ def on_inv(self, message):
                 # save txid
                 self.tx_invs_received[i.hash] += 1
 
+        super().on_inv(message)
```

Comment:
I don't see why https://github.com/bitcoin/bitcoin/pull/18807/files#diff-2dd85d481900d4ad19d113d2114861b0134bcd283435e95b18d10adf5ad381a0 couldn't be done and make 18807 complete